p2p_wallet/Scenes/Main/BankTransfer/CountryChoose/Country+ChooseItem.swift

```diff
@@ -10,4 +10,12 @@ extension Country: ChooseItemSearchableItem {
     }
 }
 
+extension Region: ChooseItemSearchableItem {
+    public var id: String { name }
```

Comment:
I guess name is wrong in this case, alpha3/2 code may be?

Reply:
in country we had to use name because there are 2 countries with the same name in the list

Reply:
Not really, we have USA states, they have the same alpha2/alpha3

Reply:
oh ok ok understood